app/screens/home/bottomTabs/Home.tsx

```diff
@@ -522,10 +530,91 @@ const Home = ({ route, navigation }: any): any => {
                 onChange={ondobChange2}
               />
             )}
-            <DailyReads />
-            <FeatureDivideArea>
+             <SearchBox>
+              <OuterIconRow>
+
+                <Pressable style={styles.pressablePadding} onPress={async (e): Promise<any> => {
+                  e.preventDefault();
+                 // await searchList(queryText);
+
+                }}>
+                  <Icon
+                    name="ic_search"
+                    size={20}
+                    color="#000"
+
+                  />
+                </Pressable>
+
+              </OuterIconRow>
+              <SearchInput
+                autoCapitalize="none"
+                autoCorrect={false}
+                clearButtonMode="always"
+                onFocus={(): any => {
+                 // setHistoryVisible(true);
+                }}
+                onChangeText={async (queryText: any): Promise<any> => {
+                  console.log('loghererer', queryText)
+                  if (queryText.replace(/\s/g, "") == "") {

```

Comment:
queryText.replace(/\s/g, "") has been used in multiple places. Consider storing the query text in a variable to avoid redundancy.